resources/js/components/Listing/Filters/CategoryFilterCategory.vue

```diff
@@ -31,12 +31,7 @@ export default {
             type: Object,
             default: function () {
                 return {
```

Comment:
This looks breaking if it has been used to style the categories, don't we want to maintain that compatibility?

Reply:
For me I think it is not necessary to keep te compatibility if we merge this in 2.x